extras/parse_travis_log.py

```diff
@@ -899,7 +900,6 @@ def build_return_code(status_vera_init,
        (status_make_install) and \
        (status_tests) and \
        (number_of_msgs_in_summary(summary_vera) == 0) and \
-       (number_of_msgs_in_summary(summary_cppcheck) == 0) and \
```

Comment:
If we decided no to use flags, then this line should stay in place, commented out, and with an accompanying comment explaining why the line is commented out and when it should be activated again.